platformio.ini

```diff
@@ -134,6 +145,20 @@ build_flags = ${env.build_flags}
   ; -D DEBUG=1 -D CORE_DEBUG_LEVEL=1 -D ARDUINOJSON_DEBUG=1 ;; for more debug output
 
 
+[env:lolin_c3_mini]
+board = lolin_c3_mini ;https://github.com/platformio/platform-espressif32/blob/develop/boards/lolin_c3_mini.json
+;; platform = espressif32@5.3.0 ;; WLED default framework version
+platform = espressif32@6.3.0    ;; this one behaves better for debugging
+upload_speed = 256000
+build_unflags =
```

Comment:
Can build_unflags be removed now it is not set in [env] anymore?

Reply:
Well not strictly needed if you want to use USB for debugging, however still needed  for boards like this that also have a "real" serial UART-to-USB port. 

https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/hw-reference/esp32s2/user-guide-s2-devkitc-1.html

For most S2/-C3, the board.json includes `-DARDUINO_USB_CDC_ON_BOOT=1`, but for serial UART-to-USB you's need `-DARDUINO_USB_CDC_ON_BOOT=0`.